manifests/params.pp

```diff
@@ -12,7 +12,7 @@
   $config_dir     = '/opt/kafka/config'
   $bin_dir        = '/opt/kafka/bin'
   $log_dir        = '/var/log/kafka'
-  $mirror_url     = 'https://www.apache.org/dyn/closer.lua?action=download&filename='
+  $mirror_url     = 'https://archive.apache.org/dist/'
```

Comment:
I think upgrading to the latest version is a better thing. I don't think archive is mirrored so it's not very nice to default to it. I don't mind updating the default version and probably would even encourage it but that does deserve a changelog entry. As I said: opening an issue is the best way to do so.

Reply:
I disagree. This could simply make the tests fail tomorrow whenever "apache" believes it's okay to remove the old versions. This happens to zookeeper as well and 3.6.1 is not very old, but still gone.

Reply:
Fair enough, but I would consider it something that must be noted in the changelog. It changes default behavior and as a user, that's something of note. Remember that modulesyncs are left out of the changelog by default and we don't want to require users to read the git log. That's why this is important to get right.

Reply:
Okay, my suggestion:

* Merge this PR to get "green test" base again
* Update to kafka 2.7.0
* Move back to old mirror_url again
* Optionally: Use the archive in testing, not sure how this works.

I can carry out 1-3 if you agree.

Reply:
I don't understand why opening an issue that states the new behavior and linking it in this PR is too hard. It is trivial and solves the problem.

Reply:
Opened the issue and cross referenced.

Reply:
Thanks!